packages/eddsa-pcd/src/EdDSAPCD.ts

```diff
@@ -175,11 +174,15 @@ export const EdDSAPCDPackage: PCDPackage<
 };
 
 export async function getEdDSAPublicKey(
-  privateKey: string
+  privateKey: string | Uint8Array
```

Comment:
Which is it? A string or a Uint8Array? How am I meant to compare them if I want to verify that a PCD is signed by a known public key?